contracts/src/tokens/drop/IDropFacet.sol

```diff
@@ -0,0 +1,86 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.23;
+
+// interfaces
+
+// libraries
+
+// contracts
+
+interface IDropFacetBase {
+  /// @notice A struct representing a claim condition
+  /// @param startTimestamp The timestamp at which the claim condition starts
+  /// @param maxClaimableSupply The maximum claimable supply for the claim condition
+  /// @param supplyClaimed The supply already claimed for the claim condition
+  /// @param merkleRoot The merkle root for the claim condition
+  /// @param currency The currency to claim in
+  /// @param penaltyBps The penalty in basis points for early withdrawal
+  struct ClaimCondition {
+    uint256 startTimestamp;
+    uint256 endTimestamp;
+    uint256 maxClaimableSupply;
+    uint256 supplyClaimed;
+    bytes32 merkleRoot;
+    address currency;
+    uint256 penaltyBps;
```

Comment:
```suggestion
    address currency;
    uint40 startTimestamp;
    uint40 endTimestamp;
    uint16 penaltyBps;
    uint256 maxClaimableSupply;
    uint256 supplyClaimed;
    bytes32 merkleRoot;
```
`uint16` is large enough for `10000`. `uint40` is large enough to store 34842 years. The first four variables can be packed into one slot.